lib/tasks/backfill_new_jsonb_columns.rake

```diff
@@ -0,0 +1,112 @@
+desc "Migrate access_limits json columns"
+task access_limits: :environment do
+  puts "starting access limits"
+  acc_lim_sql = "UPDATE access_limits SET temp_users = users, temp_organisations = organisations;"
+  ActiveRecord::Base.connection_pool.with_connection { |con| con.exec_query(acc_lim_sql) }
+end
+
+desc "Migrate editions json columns"
+task editions: :environment do
+  record = Edition.find_by(id: 704530)
+  details = record.details.to_json.gsub(/\\u0000/, "")
+  details = JSON.parse(details).symbolize_keys
+  record.update(details: details)
+
+  low = 0
+  high = 1000
+  total = Edition.maximum("id")
+
+  sql = "UPDATE editions SET temp_details = details, temp_routes = routes, " \
+        "temp_redirects = redirects WHERE id > #{low} AND id <= #{high};"
+
+  while high < total
+    ActiveRecord::Base.connection_pool.with_connection { |con| con.exec_query(sql) }
+    puts "finished editions batch #{low} to #{high}"
+    low += 1000
+    high += 1000
+  end
+end
+
+desc "Migrate events json columns"
+task events: :environment do
+  low = 0
+  high = 1000
+  total = Event.maximum("id")
+
+  sql = "UPDATE events SET temp_payload = payload WHERE id > #{low} AND id <= #{high};"
+
+  while high < total
+    Event.find_each(start: low, finish: high) do |event|
+      next if event.payload.nil?
+
+      payload = event.payload.to_json.gsub(/\\u0000/, "")
+      payload = JSON.parse(payload).symbolize_keys
+      event.update(payload: payload)
+    end
```

Comment:
Do we still need to do this? I thought it was only one `Edition` that had a null byte in the string?

Reply:
There's at least one event with null bytes (id 31148138)

Reply:
Hmm... I wonder if that event is the event that caused the edition with the null byte.

I think it would be better to deal with these similar to how we dealt with the edition with a special case, because then it's easier to see exactly what data we're changing and there is less of a risk of changing something accidentally. (Plus it would make the task a slightly bit faster).

Reply:
I've changed this now so we only remove the one event and edition